packages/esbuild-plugin/src/index.ts

```diff
@@ -38,34 +38,42 @@ function esbuildReleaseInjectionPlugin(injectionCode: string): UnpluginOptions {
 
 function esbuildDebugIdInjectionPlugin(): UnpluginOptions {
   const pluginName = "sentry-esbuild-debug-id-injection-plugin";
-  const proxyNamespace = "sentry-debug-id-proxy";
   const stubNamespace = "sentry-debug-id-stub";
 
   return {
     name: pluginName,
 
     esbuild: {
       setup({ onLoad, onResolve }) {
-        onResolve({ filter: /.*/ }, (args) => {
+        onResolve({ filter: /.*/, namespace: "file" }, (args) => {
           if (args.kind !== "entry-point") {
             return;
           }
+
           return {
             pluginName,
-            path: args.path,
-            namespace: proxyNamespace,
+            // needs to be an abs path, otherwise esbuild will complain
+            path: path.isAbsolute(args.path) ? args.path : path.join(args.resolveDir, args.path),
+            namespace: "file", // passthrough
             pluginData: {
+              isProxyResolver: true,
               originalPath: args.path,
               originalResolveDir: args.resolveDir,
             },
           };
         });
 
-        onLoad({ filter: /.*/, namespace: proxyNamespace }, (args) => {
+        onLoad({ filter: /.*/, namespace: "file" }, (args) => {
```

Comment:
also here, do we need `namespace` here?

Reply:
Yeah this namespace is also unnecessary. I think I defined it because I wanted to be conservative over what files we resolve/load, but thinking about it again, it seems unnecesary.

Thanks for questioning this!

Reply:
29f39f630cdb3de6865a6c627a1ec629e2b53b5f